docs/tutorials/reuse-ssh-connections.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reuse SSH Connections
```

Comment:
All (or most) of our other tutorials use the "ing" form of the verb, so "Reusing SSH Connections" would match better with other tutorials. The file name should be changed to match as well.